automation/roles/cloud_resources/defaults/main.yml

```diff
@@ -32,6 +32,9 @@ database_public_allowed_ips: "" # (comma-separated list of IP addresses in CIDR
 
 # Load balancer
 cloud_load_balancer: true # Create a Load Balancer in the Cloud.
+cloud_load_balancer_unified: false # Creates a single load balancer with different ports
+cloud_load_balancer_replica_port: 5433
+cloud_load_balancer_sync_port: 5434
```

Comment:
Maybe we should use the same ports as we do when using [haproxy](https://github.com/vitabaks/autobase/blob/master/automation/roles/common/defaults/main.yml#L25)?

```yaml
  master: 5000
  replicas: 5001
  replicas_sync: 5002
  replicas_async: 5003
```